python/server/auto_generated/qgsconfigcache.sip.in

```diff
@@ -53,10 +61,37 @@ value is ``False``).
 .. versionadded:: 3.0
 %End
 
+    QString strategyName() const;
+%Docstring
+Returns the name of the current strategy
+
+.. versionadded:: 3.24
```

Comment:
3.26 ?
For all other `3.24` occurrences in the PR

Reply:
Yes target is now 3.26

Reply:
Fixed target version